src/structures.jl

```diff
@@ -55,17 +55,77 @@ end
 mutable struct TulipaConstraint
     indices::DataFrame
     table_name::String
+    num_rows::Int
+    constraint_names::Vector{Symbol}
     expressions::Dict{Symbol,Vector{JuMP.AffExpr}}
+    duals::Dict{Symbol,Vector{Float64}}
 
     function TulipaConstraint(connection, table_name::String)
         return new(
             DuckDB.query(connection, "SELECT * FROM $table_name") |> DataFrame,
             table_name,
+            only([ # only makes sure that a single value is returned
+                row.num_rows for
+                row in DuckDB.query(connection, "SELECT COUNT(*) AS num_rows FROM $table_name")
+            ]), # This loop is required to access the query resulted, because it's a lazy struct
```

Comment:
Maybe it's a silly question, but why can it return more than one value? I would expect it to always return only one value, although the `only` is a safeguard to ensure that.